docs/user/05-60-kyma-bindings.md

```diff
@@ -16,32 +31,87 @@ X-Broker-API-Version: 2.14
 }
 ```
 
-The Broker returns a kubeconfig with a JWT token used as a user authentication mechanism. The token is generated using Kubernetes TokenRequest attached to a ServiceAccount, ClusterRole, and ClusterRoleBinding, all named `kyma-binding-{{binding_id}}`. Such an approach allows for modifying the permissions granted to the kubeconfig.
-Besides the kubeconfig, there is metadata in the response with the **expires_at** field, which specifies the expiration time of the kubeconfig. 
-To specify the duration for which the generated kubeconfig is valid, provide the **expiration_seconds** in the `parameter` object of the request body.
-
-| Name                   | Default | Description                                                                                                                                                                                                                                                                                                                                                          |
-|------------------------|---------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| **expiration_seconds** | `600`   | Specifies the duration (in seconds) for which the generated kubeconfig is valid. If not provided, the default value of `600` seconds (10 minutes) is used, which is also the minimum value that can be set. The maximum value that can be set is `7200` seconds (2 hours).                                             |
+If a binding is successfully created, the endpoint returns `201 Created` or `200 OK` status code depending on if the curret request created the binding or it already existed before.
 
-## Fetching a Service Binding 
+### Fetching a Service Binding 
 
-To fetch a binding, use a GET request to the Broker API:
+To fetch a binding, use a GET request to KEB API:
 
 ```
 GET http://localhost:8080/oauth/v2/service_instances/{{instance_id}}/service_bindings/{{binding_id}}
 X-Broker-API-Version: 2.14
 ```
 
-The Broker returns the `200 OK` status code with the kubeconfig in the response body. If the binding does not exist, the instance does not exist, or the instance is suspended, the Broker returns a `404 Not Found` status code.
+KEB returns the `200 OK` status code with the kubeconfig in the response body. If the binding does not exist, the instance does not exist, or the instance is suspended, KEB returns a `404 Not Found` status code.
 
-## Unbinding
+All the codes are based on the [Open Service Broker API specification](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#fetching-a-service-binding) 
```

Comment:
All http codes. 
And this is not entirely true.

Reply:
"based" does not mean "exactly the same". Every time we are discussing the codes we are referring to OSB API specification.